x-pack/plugins/security_solution/cypress/tasks/login.ts

```diff
@@ -323,24 +324,42 @@ export const waitForPage = (url: string) => {
   );
 };
 
-export const visit = (
-  url: string,
-  onBeforeLoadCallback?: (win: Cypress.AUTWindow) => void,
-  role?: ROLES
-) => {
-  cy.visit(
-    `${
-      role ? getUrlWithRoute(role, url) : url
-    }?timerange=(global:(linkTo:!(timeline),timerange:(from:1547914976217,fromStr:'2019-01-19T16:22:56.217Z',kind:relative,to:1579537385745,toStr:now)),timeline:(linkTo:!(global),timerange:(from:1547914976217,fromStr:'2019-01-19T16:22:56.217Z',kind:relative,to:1579537385745,toStr:now)))`,
-    {
-      onBeforeLoad(win) {
-        if (onBeforeLoadCallback) {
-          onBeforeLoadCallback(win);
-        }
-        disableNewFeaturesTours(win);
+export const visit = (url: string, options: Partial<Cypress.VisitOptions> = {}, role?: ROLES) => {
+  const timerange = encode({
+    global: {
+      linkTo: ['timeline'],
+      timerange: {
```

Comment:
It may be worth extracting the `timerange` object into a variable and spreading it to ensure both `global` and `timeline` are using the same config.